### ISSUE TITLE
how to use circle_loss an cross entropy loss ,alpha =128 is too big

### ISSUE THREAD

**Issue by yulinhuyang** [JDAI-CV/fast-reid#326]
## ❓ How to do something using fastreid

Describe what you want to do, including:

1. what inputs you will provide, if any:
2. what outputs you are expecting:

NOTE:

1. Only general answers are provided.
   If you want to ask about "why X did not work", please use the
   [Unexpected behaviors](https://github.com/JDAI-CV/fast-reid/issues/new/choose) issue template.

2. About how to implement new models / new dataloader / new training logic, etc., check documentation first.

3. We do not answer general machine learning / computer vision questions that are not specific to fastreid, such as how a model works, how to improve your training/make it converge, or what algorithm/methods can be used to achieve X.

when I try to use circle_loss ，alpha =128，it result in the loss is too big。it bigger than the CE loss. what is appropriate value of alpha
